samples/bluetooth/direction_finding_central/README.rst

```diff
@@ -0,0 +1,213 @@
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns
+
+The sample also requires an antenna matrix when operating in angle of arrival mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix, or any other antenna matrix.
+
+Overview
+********
+
+The direction finding central sample application uses Constant Tone Extension (CTE), received in a request response from a connected peer device.
+
+The sample supports two direction finding modes:
+
+* angle of arrival (AoA)
+* angle of departure (AoD)
+
+By default, both modes are available in the sample.
+
+Configuration
+*************
+
+|config|
+
+This sample configuration is split into the following two files:
+
+* generic configuration is available in :file:`prj.conf` file
+* board specific configuration is available in :file:`boards/<BOARD>.conf` file
+
+Board specific configuration involves configuring the Bluetooth LE controller.
+For :ref:`nRF5340 DK <ug_nrf5340>`, the Bluetooth LE controller is part of a ``child image`` aimed to run on the network core.
+Configuration for the child image is stored in :file:`child_image/` subdirectory.
+
+Angle of departure mode
+=======================
+
+To build this sample with angle of departure mode only, set ``OVERLAY_CONFIG`` to :file:`overlay-aod.conf`.
+
+See :ref:`cmake_options` for instructions on how to add this option.
+For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
+
+To build this sample for :ref:`nRF5340 DK <ug_nrf5340>`, with angle of arrival mode only, add content of :file:`overlay-aod.conf` file to :file:`child_image/hci_rpmsg.conf` file.
+
+Antenna matrix configuration for angle of arrival mode
+======================================================
+
+To use this sample when angle of arrival mode is enabled, additional configuration of GPIOs is required to control the antenna array.
+Example of such configuration is provided in a devicetree overlay file :file:`nrf52833dk_nrf52833.overlay`.
+
+The overlay file provides the information about which GPIOs should be used by the Radio peripheral to switch between antenna patches during the CTE reception in the AoA mode.
+At least two GPIOs must be provided to enable antenna switching.
+
+The GPIOs are used by the Radio peripheral in order given by the ``dfegpio#-gpios`` properties.
+The order is important because it affects mapping of the antenna switching patterns to GPIOs (see `Antenna patterns`_).
+
+To successfully use the direction finding central when the AoA mode is enabled, provide the following data related to antenna matrix design:
+
+* Provide the GPIO pins to ``dfegpio#-gpios`` properties in :file:`nrf52833dk_nrf52833.overlay` file
+* Provide the default antenna that will be used to receive PDU ``dfe-pdu-antenna`` property in :file:`nrf52833dk_nrf52833.overlay` file
```

Comment:
```suggestion
* Provide the default antenna that will be used to receive the PDU ``dfe-pdu-antenna`` property in :file:`nrf52833dk_nrf52833.overlay` file
```

or

```suggestion
* Provide the default antenna that will be used to receive PDU ``dfe-pdu-antenna`` properties in :file:`nrf52833dk_nrf52833.overlay` file
```

Reply:
@mia-ko should it be corrected?

Reply:
I've changed the line to:
`Provide the default antenna that will be used to transmit a PDU ``dfe-pdu-antenna`` property in the :file:`nrf52833dk_nrf52833.overlay` file`

`a PDU` - cause one of many
`property` there is a single `dfe-pdu-antenna` property in the overlay file.